src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,59 @@
-export const Pagination = () => {};
+import type { MouseEvent, FC } from 'react';
+import { StepButton } from '../StepButton';
+import { PageNumber } from '../PageNumber';
+
+type Props = {
+  currentPage: number;
+  pages: number;
+  onPageChange: (action: number | string) => void;
+};
+
+export const Pagination: FC<Props> = ({ currentPage, onPageChange, pages }) => {
+  const pageNumbers = Array.from({ length: pages }, (_, i) => i + 1);
+
+  const nextClick = (e: MouseEvent) => {
+    e.preventDefault();
+
+    if (currentPage === pages) {
```

Comment:
from its name (`pages`) I expect the object to be there, but it is actually a number